api/v1/configurationpolicy_types.go

```diff
@@ -340,6 +342,19 @@ func (c ComplianceType) IsMustNotHave() bool {
 // +kubebuilder:validation:Enum=MustHave;Musthave;musthave;MustOnlyHave;Mustonlyhave;mustonlyhave
 type MetadataComplianceType string
 
+// RecreateOption describes the condition when to delete and recreate an object when an update is required. IfRequired
```

Comment:
@dockerymick could you please review this text?

Reply:
Could you add default behaviour?